src/request/change-password/ChangePassword.js

```diff
@@ -0,0 +1,211 @@
+/* global Nimiq */
+/* global Constants */
+/* global PassphraseBox */
+/* global PassphraseSetterBox */
+/* global KeyStore */
+/* global Errors */
+/* global Utf8Tools */
+/* global TopLevelApi */
+/* global DownloadLoginFile */
+/* global LoginFileIcon */
+/* global ProgressIndicator */
+/* global Iqons */
+/* global LoginFile */
+
+/**
+ * @callback ChangePassword.resolve
+ * @param {KeyguardRequest.SimpleResult} result
+ */
+
+class ChangePassword {
+    /**
+     * @param {Parsed<KeyguardRequest.SimpleRequest>} request
+     * @param {ChangePassword.resolve} resolve
+     * @param {reject} reject
+     */
+    constructor(request, resolve, reject) {
+        this._resolve = resolve;
+        this._request = request;
+        this._reject = reject;
+
+        /** @type {Key} */
+        // eslint-disable-next-line no-unused-expressions
+        this._key;
+
+        // Pages
+        /** @type {HTMLFormElement} */
+        this.$enterPassword = (document.getElementById(ChangePassword.Pages.ENTER_PASSWORD));
+        /** @type {HTMLFormElement} */
+        this.$setPassword = (document.getElementById(ChangePassword.Pages.SET_PASSWORD));
```

Comment:
The `$enterPassword` and `$setPassword` page variables are only used within the constructor, thus they don't need to be assigned to the instance but can be local `const` only.